packages/backend/src/plugins/search.ts

```diff
@@ -96,5 +96,6 @@ export default async function createPlugin({
   return await createRouter({
     engine: indexBuilder.getSearchEngine(),
     logger,
+    discovery,
```

Comment:
This might be contentious but I needed a reasonable default baseUrl to assume for locations which are a path rather than a full url (by default they all are AFAIK). Thinking about it I could just hard-code some localhost url if we'd rather not add a dependency just for that reason.